src/Components/Nav/components/NavAside.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styled from 'styled-components';
+import NavAsideIcon from './NavAsideIcon';
+
+import { BiCart } from 'react-icons/bi';
+import { BsBookmark, BsBell } from 'react-icons/bs';
+import { CgProfile } from 'react-icons/cg';
+import { AiOutlineDown } from 'react-icons/ai';
+import { BiSearch } from 'react-icons/bi';
+import { MdPhotoCamera } from 'react-icons/md';
+
+const TEST = [BsBookmark, BsBell, BiCart, CgProfile];
+
+function NavAside() {
+  return (
+    <NavMain>
+      <div>
+        <form className="form">
```

Comment:
```suggestion
        <Form>
```
- 클래스명을 쓰고 싶어질 때는 대체로 스타일드 컴포넌트로 분리하기 좋은 타이밍입니다

Reply:
최대한 클래스명 쓰지않고 컴포넌트로 분리하여 사용하겠습니다.